sparse/test/local_test.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"bytes"
+	"os"
+
+	"testing"
+
+	. "github.com/longhorn/sparse-tools/sparse"
+)
+
+func TestSyncLocalFile(t *testing.T) {
+	layoutFrom := []FileInterval{
+		{Kind: SparseData, Interval: Interval{Begin: 0, End: 1 * Blocks}},
+		{Kind: SparseHole, Interval: Interval{Begin: 1 * Blocks, End: 2 * Blocks}},
+		{Kind: SparseData, Interval: Interval{Begin: 2 * Blocks, End: 3 * Blocks}},
+	}
```

Comment:
Can we add more test cases

1. data-hole-data (you've added)
2. hole-data-hole
3. hole-data
4. data-hole

Thank you.